tests/instruments/agilent/test_agilent34450A.py

```diff
@@ -76,28 +70,35 @@ def make_reseted_dmm(self):
                           [1E-4, 1E-4], [1E-3, 1E-3], [1E-2, 1E-2], ["MIN", 1E-9], ["MAX", 1E-2],
                           ["DEF", 1E-6]]
 
+    DMM = Agilent34450A(RESOURCE)
+
+    ############
+    # FIXTURES #
+    ############
+    @pytest.fixture
+    def make_reseted_dmm(self):
+        self.DMM.reset()
+        return self.DMM
+
     #########
     # TESTS #
     #########
 
     def test_dmm_initialization_bad(self):
         bad_resource = "USB0::10893::45848::MY12345678::0::INSTR"
-        with pytest.raises(VisaIOError):
+        # The pure python VISA library (pyvisa-py) raises a ValueError while the PyVISA library raises a VisaIOError.
```

Comment:
huh, weird, that inconsistency might be worth to report a bug against pyvisa-py...